hummingbird/ml/operator_converters/_normalizer_implementations.py

```diff
@@ -0,0 +1,30 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Base classes for normalizer implementations.
+"""
+
+import torch
+
+from ._base_operator import BaseOperator
+
+
+class Normalizer(BaseOperator, torch.nn.Module):
```

Comment:
Can you please add a one linear of documentation? Some like: Class implementing Normalizer operators in PyTorch. Supported normalizers are L1, L2 and Max.